README.md

```diff
@@ -5,8 +5,8 @@ pajbot is a twitch chat bot created by [pajlada](http://twitch.tv/pajlada).
 
 ## Quick install
 
-1. Install library requirements by typing `pip install -r requirements.txt` in
-   the root folder
+1. Install library requirements by typing `./scripts/venvinstall.sh` in
+   the root pajbot folder
```

Comment:
I don't really agree with adding `pajbot` here

Reply:
Yeah I agree, adding `pajbot` here makes it seem like you need to cd into the `pajbot` folder first